### PR TITLE
Enhance Documentation & Development Experience

### LINKED ISSUE #14: Project documentation on GitHub
The goal of this issue is create the repo documentation:
* README.md
* CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Michael-Obele**
## Pull Request Information

**What kind of change does this PR introduce?**

Documentation improvements and developer experience enhancements.

**Issue Number:**

*Closes #14: Project documentation on GitHub. This pull request introduces a CONTRIBUTING guide as requested in issue #14.

**Screenshots/videos:**

N/A (This pull request focuses on documentation and configuration changes.)

**If relevant, did you update the documentation?**

Yes, this pull request introduces a new `CONTRIBUTING.md` file and updates the `README.md` file.

**Summary**

This pull request improves the project's documentation and development experience in several ways:

* **Comprehensive CONTRIBUTING Guide:** Provides clear instructions for contributing to the project, including reporting bugs, suggesting features, and making pull requests. 
* **Enhanced README:** Expands the `README.md` to offer a more thorough introduction to the project and simplifies local development setup.
* **Improved Developer Experience:** Reduces unnecessary spell check warnings for Visual Studio Code users.

These changes aim to empower contributors, improve project understanding, and streamline development workflows.
 
**Does this PR introduce a breaking change?**

No, this pull request does not introduce any breaking changes.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (DhairyaMajmudar) [inline .vscode/settings.json:1]: Hi @Michael-Obele 
any specific reason for this change ?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+Before contributing, please familiarize yourself with our [Code of Conduct](https://github.com/json-schema-org/.github/blob/main/CODE_OF_CONDUCT.md).
+
+
+##  How to Contribute 🚀
+
+There are several ways you can contribute to the JSON Schema website:
+
+* [**Reporting Bugs** 🐞](#reporting-bugs-🐞)
+* [**Suggesting Features** 💡](#suggesting-features-💡)
+* [**Improving the Website** 🌐](#improving-the-website-🌐)
```
Comment (DhairyaMajmudar) [inline CONTRIBUTING.md:27]: Here how's about adding one more section related to docs improvement.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+**Limited Website Functionality Changes:**
+
+The website is built with a tool called CNCF Landscape, designed for managing this type of information. While large code edits aren't possible, some customization is achievable through configuration and themes.
+
+* Have an idea for a new feature that works within CNCF Landscape's limitations? Open an issue to discuss your proposal. We use a project board to track website contribution tasks: link to project board: [https://github.com/orgs/json-schema-org/projects/19](https://github.com/orgs/json-schema-org/projects/19).
+
+**Getting Started with Content or Design:**
+
+These steps are for improvements that don't involve code changes:
+
```
Comment (DhairyaMajmudar) [inline CONTRIBUTING.md:55]: Pls. fix the numbering here.

**Review by DhairyaMajmudar — CHANGES_REQUESTED** [review]

**Comment by DhairyaMajmudar** [comment]
Hello @Michael-Obele left some changes, pls. once have a look on them : )

File: .vscode/settings.json (hunk 1 above)
Comment (Michael-Obele) [reply .vscode/settings.json:1]: This was supposed to be an internal setting for my VSCode; I will remove it.

File: CONTRIBUTING.md (hunk 3 above)
Comment (Michael-Obele) [reply CONTRIBUTING.md:55]: Hi, @DhairyaMajmudar! 
In Markdown formatting, lists automatically adjust their numbering. You can check this by using the markdown preview mode, which should show the correct numbering.

File: CONTRIBUTING.md (hunk 2 above)
Comment (Michael-Obele) [reply CONTRIBUTING.md:27]: Okay, I will do just that. Thanks

**Comment by Michael-Obele** [comment]
> Hello @Michael-Obele left some changes, pls. once have a look on them : )

Hi @DhairyaMajmudar, Sure thing. I just made some changes; if you have time, please take a look and let me know if there is anything else I should change.

**Review by DhairyaMajmudar — APPROVED** [review]

File: README.md [hunk 4]
```diff
@@ -1 +1,86 @@
...
+
+* Install Landscape2 first. Find pre-built versions for your system on the Landscape2 releases page: [https://github.com/cncf/landscape2](https://github.com/cncf/landscape2)
+
+**Building the Website:**
+
+The build process creates the website from data files. Run this command in your terminal:
+
+```bash
+landscape2 build \
+  --data-file data.yml \
```
Comment (aialok) [inline README.md:39]:
```suggestion
  --data-file landscape.yml \
```

we are changing data.yml to landscape.yml #43 
Better fix this now.

Thank you  : )

**Review by aialok — APPROVED** [review]

**Review by benjagm — APPROVED** [review]
Great job. Thanks!!

**Pull request merged**